Dfe.Academies.External.Web/CypressTests/cypress.config.js

```diff
@@ -0,0 +1,19 @@
+const { defineConfig } = require('cypress')
+/*const {
+  username,
+  password
+} = require('./config')
+*/
+
+
+module.exports = defineConfig({
+  e2e: {
+    url: 'https://s184t01-a2bextcdnendpoint-ezfhhdbpembpanh5.z01.azurefd.net',
```

Comment:
⚠️ As mentioned above, these all need to be removed from the code before merge and become environment variables